opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -5164,7 +5158,7 @@ interface Location {
 input LocationAddInput {
   stix_id: StixId
   x_opencti_stix_ids: [StixId]
-  type: LocationType!
+  type: String!
```

Comment:
Why this change ?

Reply:
LocationType has been removed from schema

Reply:
Done because of graphQL enum not supporting the "-" character...